src/style.css

```
@@ -1,3 +1,60 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+header {
```

Use only class selector
```suggestion
.header {
```